src/nlsolve/muller.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    SimpleMuller()
+
+Muller's method.
```

Comment:
Make the documentation a bit more detailed. References, restrictions like only scalars and such